testapp/index.php

```diff
@@ -28,14 +29,37 @@ function (Request $req) {
         /** @var string $content */
         $content = $req->getContent(false);
         if (!empty($content)) {
-            $data = json_decode($content, true);
-            if (!is_array($data)) {
-                $ret['content'] = $content;
-            } else {
-                foreach ($data as $key => $value) {
-                    $ret[$key] = $value;
+
+            if (0 === strpos($req->headers->get('Content-Type'), 'application/json')) {
```

Comment:
could be simpler: ``if ('json' === $request->getContentType())``. the Request object has a ``getContentType`` returning the Symfony format associated to the content type